### DISCUSSION TITLE
Size atribute has no effect

### DISCUSSION THREAD

**Discussion by mahxds** [Budibase/budibase#7690] [Help] [answered]
Hi all, im new to Budibase and really enjoying it so far.

currently im struggling with the size attribute of components. Somehow i can't manage it to set them to a fixed size.

i tried different approaches  like putting the components in a container, in a form, then in a container and and.

No matter where i change the size (width) of a component, it wont have a effect in the preview or published version.

what am i doing wrong?

**Comment by mahxds** [comment]
nobody an idea what im doing wrong :D

**Comment by eftomi** [comment, marked as answer]
Hi, you have to use units of measure in width, height and other properties, e.g. pixels as in 150 px or 35 % - percentages relate to the parent element's size and are usually better if you have to respect size responsiveness of your app. In parent elements you can use alignment settings.

https://www.w3schools.com/cssref/css_units.asp

**Reply by mahxds** [reply]
THANKS ALOT!  SOOO Easy yet i didnt see it anywhere , thanks alot :)
